### ISSUE TITLE
[Enhancement] Focus State

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3840]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Focus color is not accessible

Consider removing focus indicator when using mouse.
Consider inside stroke focus.

## Describe the solution you'd like
Change focus color to: #238CEC
Focus Inner shadow sm (used on small components. Radio, Checkbox and Toggle)
Focus Inner shadow md (default)
Focus shadow md (Only used on links)

## Have you considered any alternatives?
We could remove the "Focus Inner shadow sm" and use the "Focus shadow md". The naming is up to debate

## Are there any additional context?
https://www.figma.com/design/bTihgLkrfmI3AkiaeEuUCF/Accessibility-design-assessment?m=auto&node-id=2233-159&t=SjerN0oTA76IbyuL-1

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by Fuzzy3** [comment]
### Style
1) Focus Inner Shadow SM (inset): `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 3px var(--System-background-color, #ffffff) inset;`
2) Focus Inner Shadow MD (inset): `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 4px var(--System-background-color, #ffffff) inset;`
3) Focus Outer Shadow MD: `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 4px var(--System-background-color, #ffffff);`
